ncbi-submit/config/ncbi_submit_properties.py

```diff
@@ -0,0 +1,72 @@
+base_script_dir = "/root/"
+templates_dir = base_script_dir + "templates"
+schemas_dir = base_script_dir + "schemas/"
+private_key_path = base_script_dir + ".ssh/id_rsa_ncbi"
+
+# The URL to use when validating the submission XML.
+validation_url = "https://www.ncbi.nlm.nih.gov/projects/biosample/validate/"
+
+# ASCP command and default options
+ascp_cmd = [
+    base_script_dir + ".aspera/connect/bin/ascp",
+    "-k2",
+    "-L", "logs",
+    "--file-manifest-path=logs",
+    "--file-manifest=text"
+]
+
+schema_paths = {
+    'submission': schemas_dir + 'submission.xsd',
+    'bioproject': schemas_dir + 'bioproject.xsd',
+    'biosample':  schemas_dir + 'biosample.xsd',
+    'genome':     schemas_dir + 'genome.xsd'
+}
+
+ncbi_user = "asp-bioci"
+ncbi_host = "upload.ncbi.nlm.nih.gov"
+# ncbi_sumbit_path = "submit/Production"
+ncbi_sumbit_path = "submit/Test"
```

Comment:
Will do. I'm waiting until we actually create the production version of the utility.